calc/token.py

```diff
@@ -31,13 +38,22 @@ class Token:
 
     Rabbit hole:
         The ``type`` keyword is a reserved word in Python. It is typically bad
-        practice to create a new variable that overrides a reserved word, but it
-        is useful in this context and the scope is limited, so we take advantage
-        of that here.
+        practice to create a new variable that overrides a reserved word, but
+        it is useful in this context and the scope is limited, so we take
+        advantage of that here.
     """
 
+    # pylint: disable=redefined-builtin
     def __init__(self, type, value):
-        """Constructor for a :class:`Token` object."""
+        """Constructor for a :class:`Token` object.
+
+        args:
+            type: A name used to associate what kind and parameters of data
```

Comment:
Google docstrings (especially those parsed by [`sphinx.ext.napoleon`](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)) say that the `__init__` method for a class can be documented at the class level or on the `__init__` method itself, but that the styles should not be mixed. The arguments to `__init__` have alread been documented above by the
```
"""
Args:
    ...
"""
```
section. Additionally, the type information has been left out here.